process-controller/src/main/java/org/jboss/as/process/ProcessController.java

```diff
@@ -342,7 +340,9 @@ void sendInventory() {
                         StreamUtils.writeInt(os, processCollection.size());
                         for (ManagedProcess process : processCollection) {
                             StreamUtils.writeUTFZBytes(os, process.getProcessName());
-                            os.write(process.getAuthKey().getBytes(StandardCharsets.US_ASCII));
+                            // TODO Do we really need to send this?  The HostController generates new tokens anyway
```

Comment:
Do we need any follow up issues created for any of the TODOs in this PR?

Reply:
@fjuma I am not sure it feels like one of this issues that if we create it we will never come back to it - this was more acknowledging this is redundant but I think it has been redundant for a decade !

Reply:
Ok that's fine then :)